schema.sql

```diff
@@ -0,0 +1,15 @@
+CREATE DATABASE IF NOT EXISTS bookshelf;
+
+USE bookshelf;
+
+CREATE TABLE IF NOT EXISTS books (
+  id int NOT NULL AUTO_INCREMENT,
+  title varchar(50) NOT NULL,
+  author varchar (50) NOT NULL,
+  author_details varchar(2000) NOT NULL,
+  description varchar(2000) NOT NULL,
+  published_year int NOT NULL,
+  PRIMARY KEY (ID)
+);
```

Comment:
this is actually one of my tech debt/refactor items. I spent a long time yesterday trying to set up a foreign key so there would be a relationship between the two tables. I wasn't able to get it to work after a few hours and decided I should make it one table for now so that I could move on and not fall behind. But if you are able to help me figure out how to set it up as two tables and form the relationship between them I would appreciate it. I agree it would be better to set it up this way, I just couldn't get it to work!